src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -33,6 +33,12 @@
     MPIDI_OFI_AV(&MPIDIU_get_av((avtid), (lpid))).dest
 
 #define MPIDI_OFI_WIN(win)     ((win)->dev.netmod.ofi)
+
+#define MPIDI_OFI_MPI_ACCU_OP_INDEX(op, op_index) do {   \
+    if (op == MPI_OP_NULL) op_index = 14;                \
```

Comment:
I forgot move the original comment to here, just moved. Based on the original comment, `14` is temporarily hard coded. I think it might be fine to leave it as is, till we fix it correctly (e.g., predefine a table ? )